ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -1390,6 +1390,21 @@ vlan_interfaces:
     isis_network_point_to_point: < boolean >
     mtu: < mtu >
     no_autostate: < true | false >
+    # New improved "vrrp" data model to support multiple IDs
+    vrrp_ids:
+      - id: < vrid >
+        priority: < instance_priority >
```

Comment:
On EOS, this is referred as "priority-level". Should we use priority_level here for matching?

Reply:
Good catch Xavi. I think we should make the new model match EOS closer, so yes.